brain_api/main.py

```diff
@@ -29,6 +30,13 @@
 logging.info(f"Using species file {species_map_filename}")
 
 
+@app.put("/initialize_wifi")
+def initialize_wifi():
+    os.system("sudo sysctl -w net.ipv4.ip_forward=1")
+    os.system("sudo iptables -t nat -A POSTROUTING -o wlan0 -j MASQUERADE")
+    os.system("sudo mkdir -p /etc/iptables")
```

Comment:
i don't think these two lines are needed. they get wiped with every boot

Reply:
They are needed, it wouldn't work without them